client/trino-jdbc/src/test/java/io/trino/jdbc/TestJdbcVendorCompatibility.java

```diff
@@ -241,69 +212,73 @@ public void testTimestampWithTimeZone()
-                TIMESTAMP_WITH_TIMEZONE,
-                sessionTimezoneId,
-                (rs, reference, column) -> {
-                    Timestamp timestampForPointInTime = Timestamp.from(
-                            ZonedDateTime.of(2018, 2, 13, 13, 14, 15, 123_000_000, ZoneOffset.ofHoursMinutes(3, 15))
-                                    .toInstant());
-
-                    assertEquals(rs.getTimestamp(column).getTime(), reference.getTimestamp(column).getTime()); // point in time
-                    assertEquals(rs.getTimestamp(column), reference.getTimestamp(column));
-                    assertEquals(rs.getTimestamp(column), timestampForPointInTime);
-
-                    // with calendar
-                    assertEquals(rs.getTimestamp(column, getCalendar()).getTime(), reference.getTimestamp(column, getCalendar()).getTime()); // point in time
-                    assertEquals(rs.getTimestamp(column, getCalendar()), reference.getTimestamp(column, getCalendar()));
-                    assertEquals(rs.getTimestamp(column, getCalendar()), timestampForPointInTime);
-                });
-
-        checkRepresentation(
-                "TIMESTAMP '2018-02-13 13:14:15.123 Europe/Warsaw'", // Trino
-                ImmutableList.of(
-                        "TIMESTAMP WITH TIME ZONE '2018-02-13 13:14:15.123 Europe/Warsaw'", // PostgreSQL
-                        "from_tz(TIMESTAMP '2018-02-13 13:14:15.123', 'Europe/Warsaw')"), // Oracle
-                TIMESTAMP_WITH_TIMEZONE,
-                sessionTimezoneId,
-                (rs, reference, column) -> {
-                    Timestamp timestampForPointInTime = Timestamp.from(
-                            ZonedDateTime.of(2018, 2, 13, 13, 14, 15, 123_000_000, ZoneId.of("Europe/Warsaw"))
-                                    .toInstant());
-
-                    assertEquals(rs.getTimestamp(column).getTime(), reference.getTimestamp(column).getTime()); // point in time
-                    assertEquals(rs.getTimestamp(column), reference.getTimestamp(column));
-                    assertEquals(rs.getTimestamp(column), timestampForPointInTime);
-
-                    // with calendar
-                    assertEquals(rs.getTimestamp(column, getCalendar()).getTime(), reference.getTimestamp(column, getCalendar()).getTime()); // point in time
-                    assertEquals(rs.getTimestamp(column, getCalendar()), reference.getTimestamp(column, getCalendar()));
-                    assertEquals(rs.getTimestamp(column, getCalendar()), timestampForPointInTime);
-                });
+        try (Connection connection = createConnection();
+                Statement statement = connection.createStatement();
+                ConnectionSetup connectionSetup = new ConnectionSetup(referenceDrivers)) {
+            checkRepresentation(
+                    connection, statement, "TIMESTAMP '1970-01-01 00:00:00.000 +00:00'", // Trino
```

Comment:
I'd either put one argument per line or format like this:
```
checkRepresentation(connection, statement,
        "TIMESTAMP...
```
but feel free to ignore

Reply:
I didn't modify this -- I just added the try block around it